src/Simulation/Simulation.jl

```diff
@@ -820,6 +820,7 @@ function _calculate_potential!( sim::Simulation{T, CS}, potential_type::UnionAll
         max_tick_distance::Union{Missing, LengthQuantity, Tuple{LengthQuantity, <:Union{LengthQuantity, AngleQuantity}, LengthQuantity}} = missing,
         max_distance_ratio::Real = 5,
         depletion_handling::Bool = false,
+        depletion_tol = 0.1,
```

Comment:
Maybe final comment: can you allow for this to also have units, so that you can pass `depletion_tol = 0.1u“V“`, using to_internal_units in the function below?
And add a short description to the docstring what it does.